<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="select_combo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>당신의 식물을</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>931</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/cat.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="textballoon">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/balloon.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>선택해주세요.</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>textballoon</zorder>
  <zorder>select_combo</zorder>
  <zorder>btn_start</zorder>
  <zorder>text1</zorder>
  <zorder>cat</zorder>
  <zorder>text2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
